code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -259,6 +267,9 @@
 			user << "\blue The walls suddenly disappear."
 	return 1
 
+/datum/spellbook_entry/item/bloodbottle/CanBuy(var/mob/living/carbon/human/user,var/obj/item/weapon/spellbook/book)
+	return ..() && !limit

```

Comment:
@TheDZD so just remove the Canbuy entry? (sorry for the question, My brain is mush lately)